.changeset/blue-seahorses-warn.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
don't think we need a changeset, the generated code remains exactly the same

Reply:
Yeah, I thought I'm touching the internals a bit, but we're confident it doesn't change anything for the consumer. Removed it now.